Ginger/GingerPluginCore/ActionsLib/IUIElementAction.cs

```diff
@@ -20,7 +20,7 @@
 namespace Amdocs.Ginger.Plugin.Core
 {
 
-    public enum eElementType
+    public enum ElementType
```

Comment:
if you must change the enums names so pls change it to "eUIElementType" and "eUILocateBy"

Reply:
Done